src/Magnum/OvrIntegration/Hmd.h

```diff
@@ -358,24 +370,110 @@ class MAGNUM_OVRINTEGRATION_EXPORT Hmd {
+            return ovr_GetString(_session, OVR_KEY_GENDER, OVR_DEFAULT_GENDER);
+        }
+
+        /** @brief Player height set in the currently active Oculus profile */
+        Float playerHeight() const {
+            return ovr_GetFloat(_session, OVR_KEY_EYE_HEIGHT, OVR_DEFAULT_PLAYER_HEIGHT);
+        }
+
+        /** @brief Eye height set in the currently active Oculus profile */
+        Float eyeHeight() const {
+            return ovr_GetFloat(_session, OVR_KEY_EYE_HEIGHT, OVR_DEFAULT_EYE_HEIGHT);
+        }
+
+        /**
+         * @brief Inter populary distance set in the currently active Oculus
+         * profile
+         */
+        Float ipd() const {
+            return ovr_GetFloat(_session, OVR_KEY_IPD, OVR_DEFAULT_IPD);
+        }
+
+        /**
+         * @brief Neck to eye distance set in the currently active Oculus
+         * profile
+         */
+        std::array<Float, 2> neckToEyeDistance() const {
+            std::array<Float, 2> values{{OVR_DEFAULT_NECK_TO_EYE_HORIZONTAL, OVR_DEFAULT_NECK_TO_EYE_VERTICAL}};
+            ovr_GetFloatArray(_session, OVR_KEY_NECK_TO_EYE_DISTANCE, values.data(), 2);
+            return values;
+        }
+
+        /**
+         * @brief State of the eye relief dial set in the currently active
+         * Oculus profile
+         */
+        Int eyeReliefDial() const {
+            return ovr_GetInt(_session, OVR_KEY_EYE_RELIEF_DIAL, OVR_DEFAULT_EYE_RELIEF_DIAL);
+        }
+
+        /**
+         * @brief Eye to node distance set in the currently active Oculus
+         * profile
+         */
+        std::array<Float, 2> eyeToNoseDistance() const {
+           std::array<Float, 2> values{{0.0f, 0.0f}};
+            ovr_GetFloatArray(_session, OVR_KEY_EYE_TO_NOSE_DISTANCE, values.data(), 2);
+            return values;
+        }
+
+        /**
+         * @brief Maximal eye to plate distance set in the currently active
+         * Oculus profile
+         */
+        std::array<Float, 2> maxEyeToPlateDistance() const {
+            std::array<Float, 2> values{{0.0f, 0.0f}};
+            ovr_GetFloatArray(_session, OVR_KEY_MAX_EYE_TO_PLATE_DISTANCE, values.data(), 2);
+            return values;
+        }
+
+        /** @brief Eye cup set in the currently active Oculus profile */
+        std::string eyeCup() const {
+            return ovr_GetString(_session, OVR_KEY_EYE_CUP, "A");
+        }
+
+        /** @brief Status of the OVR session */
+        SessionStatusFlags sessionStatus() const;
+
     private:
-        explicit Hmd(::ovrHmd hmd);
+        explicit Hmd(::ovrSession hmd);
 
-        ::ovrHmd _hmd;
+        ::ovrSession _session;
         ::ovrHmdDesc _hmdDesc;
         ovrPosef _ovrPoses[2];
         ovrVector3f _hmdToEyeViewOffset[2];
         ::ovrViewScaleDesc _viewScale;
 
-        ovrFrameTiming _frameTiming;
+        Double _predictedDisplayTime;
         ovrTrackingState _trackingState;
 
-        UnsignedInt _frameIndex;
+        long long _frameIndex;
```

Comment:
Ah, I see, thanks for pointing that out. Yes, `incFrameIndex` should probably return the same type.